### PR TITLE
Feature/doc 172

### PR DESCRIPTION
**Pull request by shikhanansi**
Hi Millie,
Please review this PR.
Regards,
Shikha

### CODE REVIEW

File: _includes/configuration/min-height.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ## min_height
 
-This option allows you to set the minimum height that a user can stretch the entire TinyMCE interface (by grabbing the dragable area in the bottom right of the editor interface) when using the modern theme.
+The **min_height** option has two kinds of behaviors depending on the state of the [`autoresize`]({{site.baseurl}}/plugins/autoresize/) plugin:
+
+* `autoresize` OFF (Default) : Without the `autoresize` plugin, this option allows you to set the minimum height that a user can stretch the entire TinyMCE interface (by grabbing the dragable area in the bottom right of the editor interface).
```
Comment (blackcathikari) [inline _includes/configuration/min-height.md]: "stretch" -> "shrink"?

File: _includes/configuration/width.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ## width
 
-Set the width of the editor in pixels.
+Set the width of the editor.
 
-**Type:** `Number`
+> Note: TinyMCE sets the height in pixels if a number is provided. However, if TinyMCE is provided a string it assumes the value is valid CSS and simply sets the editor's height as the string value. This allows for alternate units such as `%`, `em` and `vh`.
```
Comment (blackcathikari) [inline _includes/configuration/width.md]: "height" -> "width"

File: _includes/configuration/min-height.md (hunk 1 above)
Comment (shikhanansi) [reply _includes/configuration/min-height.md]: Fixed

File: _includes/configuration/width.md (hunk 2 above)
Comment (shikhanansi) [reply _includes/configuration/width.md]: Fixed

**Pull request merged**